test/L2/L2PriceFeedWithoutRounds.t.sol

```diff
@@ -3,27 +3,17 @@ pragma solidity 0.8.23;
 
 import { ERC1967Proxy } from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
 import { OwnableUpgradeable } from "@openzeppelin-upgradeable/contracts/access/OwnableUpgradeable.sol";
-import { Test } from "forge-std/Test.sol";
+import { Test, console } from "forge-std/Test.sol";
 import { L2PriceFeedWithoutRoundsFactory } from "src/L2/L2PriceFeedWithoutRoundsFactory.sol";
 import { L2PriceFeedWithoutRounds } from "src/L2/L2PriceFeedWithoutRounds.sol";
-
-contract L2PriceFeedWithoutRoundsV2Mock is L2PriceFeedWithoutRounds {
-    string public testVersion;
-
-    function initializeV2(string memory _version) public reinitializer(2) {
-        testVersion = _version;
-    }
-
-    function onlyV2() public pure returns (string memory) {
-        return "Hello from V2";
-    }
-}
+import { L2PriceFeedWithoutRoundsV2 } from "src/L2/upgraded/L2PriceFeedWithoutRoundsV2.sol";
 
 contract L2PriceFeedWithoutRoundsTest is Test {
     L2PriceFeedWithoutRounds public l2PriceFeed;
     L2PriceFeedWithoutRounds public l2PriceFeedImplementation;
 
     address public priceFeedAdapter = 0x1038999DCf0A302Cc8Eed72fAeCbf0eEBfC476b0;
+    address public newPriceFeedAdapter = 0x2038999Dcf0A302cc8eeD72fAECbF0EebFc476b1;
```

Comment:
Optional: it'd be better to rather generate deterministic addresses from the variable name string itself instead of hardcoding random addresses.
```suggestion
    address public priceFeedAdapter = vm.addr(uint256(bytes32("priceFeedAdapter")));
    address public newPriceFeedAdapter = vm.addr(uint256(bytes32("newPriceFeedAdapter")));
```

Reply:
Done.